tests/Behat/Context/FeatureContext.php

```diff
@@ -16,39 +16,77 @@ class FeatureContext extends SilverStripeContext
      */
     public function iShouldSeeAListOfVersions()
     {
-        $page = $this->getSession()->getPage();
-        $versionsTable = $page->find('css', '.history-viewer .table');
-        assertNotNull($versionsTable, 'I should see a list of versions');
+        $this->getVersions();
```

Comment:
It seems like this test is really vague. It might be better to test more specific outcomes like:

```
Then I should see version 2 in the list
And I should see version 3 in the list
And I should not see version 11 in the list
```

Reply:
You do have that assertion as well